### PR TITLE
Update for Kubernetes 1.26.6

### PR DESCRIPTION
**Pull request by yokaze**
This PR updates CKE to support k8s 1.26.

Signed-off-by: Daichi Sakaue <daichi-sakaue@cybozu.co.jp>

### CODE REVIEW

**Review by masa213f — CHANGES_REQUESTED** [review]
How did you update `sabakan/mock/{generated.go, models.go}`?
If possible, please check for updates to the auto-generated files in CI.

File: mtest/Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-VAULT_VERSION = 1.13.0
-K8S_VERSION = 1.25.10
-CONTAINERD_VERSION = 1.6.18
-CRITOOLS_VERSION = 1.26.0
+MANAGEMENT_ETCD_VERSION = 3.5.9
+VAULT_VERSION = 1.14.0
+K8S_VERSION = 1.26.6
+CONTAINERD_VERSION = 1.7.2
+CRITOOLS_VERSION = 1.27.1
 CT_VERSION = 0.9.0
```
[the remark this answers is not in the document]
Comment (yokaze) [reply mtest/Makefile]: @masa213f The version 0.9.4 of the ct does not have its binaries in the release, so we can only use v0.9.3.

File: Makefile [hunk 2]
```diff
@@ -9,6 +9,16 @@ all: test
 setup:
 	curl -fsL https://github.com/etcd-io/etcd/releases/download/v$(ETCD_VERSION)/etcd-v$(ETCD_VERSION)-linux-amd64.tar.gz | sudo tar -xzf - --strip-components=1 -C /usr/local/bin etcd-v$(ETCD_VERSION)-linux-amd64/etcd etcd-v$(ETCD_VERSION)-linux-amd64/etcdctl
 
+.PHONY: check-generate
+check-generate:
+	# gqlgen needs additional dependencies that will be cleared out with `go mod tidy`.
+	go get github.com/99designs/gqlgen@"$$(go list -m github.com/99designs/gqlgen | cut -d' ' -f2)"
```
[the remark this answers is not in the document]
Comment (masa213f) [reply Makefile:15]: In any case, we need to comment on this. Either is fine. :)

**Review by masa213f — APPROVED** [review]

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**